File: springmvc-xml-mybatis-maven/src/main/java/com/shiyan/service/UserService.java
package com.shiyan.service;

import com.shiyan.dto.UserDTO;

/**
 * @author kevinfeng
 * @date 2019-05-25
 */
public interface UserService {
    /**
     * 通过id查询用户
     * @param id
     * @return
     */
    UserDTO find(int id);
}

File: springboot-mybatis-gradle-kotlin/src/main/kotlin/com/shiyan/springboot/service/UserService.kt
package com.shiyan.springboot.service;

import com.shiyan.springboot.model.User
import com.github.pagehelper.PageInfo

/**
 *
 * @author kevinfeng
 * @date 2019-06-06
 */
interface UserService {
    // 查询所有用户
    fun findAll(): List<User>

    // 通过ID查询用户
    fun findUserById(id: Long): User

    // 分页查询用户
    fun page(pageNo: Int, pageSize: Int): PageInfo<User>
}